membership-attribute-service/app/controllers/AttributeController.scala

```diff
@@ -115,13 +115,20 @@ class AttributeController extends Controller with LoggingWithLogstashFields {
     def queryToAccountIds(response: QueryResponse): List[AccountId] =  response.records.map(_.Id)
 
     def getSubscriptions(accountIds: List[AccountId]): Future[Disjunction[String, List[Subscription[AnyPlan]]]] = {
-      def sub(accountId: AccountId): Future[List[Subscription[AnyPlan]]] = {
+      def sub(accountId: AccountId): Future[Disjunction[String, List[Subscription[AnyPlan]]]] =
         subscriptionService.subscriptionsForAccountId[AnyPlan](accountId)(anyPlanReads)
-      }
-      val maybeSubs: Future[List[Subscription[AnyPlan]]] = Future.traverse(accountIds)(id => sub(id)).map(_.flatten)
 
-      maybeSubs map { subs =>
-        if (subs.isEmpty) \/.left(s"Error getting subscriptions for identityId $identityId") else \/.right(subs)
+      val maybeSubs: Future[Disjunction[String, List[Subscription[AnyPlan]]]] = accountIds.traverseU(id => sub(id)).map(_.sequenceU.map(_.flatten))
```

Comment:
I think previously we would have flattened all the empty lists together then just lost any "erroring" ones.  This may not be a problem we actually have, but it's exactly what problem I had with [members-data-api](https://trello.com/c/6tfTRjJl/245-error-message-viewing-digital-pack-or-membership-tabs-in-mma)